css-two-column-layout/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
+  <div class="container">
+
+    <section class="top">
+      <div class="row t-row1 f-2rem">
+        <h1>SCALE, SPEED, AGILITY</h1>
+      </div>
+      <div class="row t-row2">
+        <p class="p-font">We've dedicated thousands of man hours making healthcare institutions in the Helium network
+          run 10x more efficiently and deliver an unparalleled quality of care.</p>
+      </div>
+    </section>
+
+    <section class="mid">
+      <div class="row">
+        <div class="col-1">
```

Comment:
I would recommend wrapping each of the collections together like this: 
```html
  <div class="column-half">
    <div class="col-1">
      <img src="images\software.svg" class="img">
    </div>
    <div class="col-2">
      <h3 class="caps">intuitive, <br /> smart software</h3>
      <p class="p-font">We've dedicated thousands of man hours making healthcare institutions in the Helium network
      run 10x more efficiently and deliver an unparalleled quality of care.</p>
    </div>
  </div>
```

This would keep all the content for each column grouped, and allow you to manipulate the elements easily without affecting other elements.

Reply:
done & done!
![css-two-column-layout_fix01](https://user-images.githubusercontent.com/42279790/151634988-bb4c1e14-93ec-4fc1-bc5f-3b34caaa4155.png)